--- a/build/npm/preinstall.js
+++ b/build/npm/preinstall.js
@@ -45,7 +45,7 @@ function hasSupportedVisualStudioVersion() {
 	const path = require('path');
 	// Translated over from
 	// https://source.chromium.org/chromium/chromium/src/+/master:build/vs_toolchain.py;l=140-175
-	const supportedVersions = ['2022', '2019'];
+	const supportedVersions = ['2026', '2022', '2019'];
 
 	const availableVersions = [];
 	for (const version of supportedVersions) {
